### PR TITLE
Added zwatershed

### PR DESCRIPTION
**Pull request by csinva**

### CODE REVIEW

File: recipes/zwatershed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - boost 1.61.*
+    - python
+    - setuptools >=18.0
+    - numpy x.x
+    - numpy >=1.7
```
Comment (ocefpaf) [inline recipes/zwatershed/meta.yaml]: You don't need this with the setting above. `conda-forge` only build `numpy >=1.10` BTW.

File: recipes/zwatershed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "zwatershed" %}
+{% set version = "0.5" %}
```
Comment (jakirkham) [inline recipes/zwatershed/meta.yaml]: Please update to `0.8`.

File: recipes/zwatershed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+{% set checksum = "df5b36634e38012e0fb3b2545d3bfc26" %}
```
Comment (jakirkham) [inline recipes/zwatershed/meta.yaml]: Please update to `e48b08909ad95785d9adfee0d1f8d318`.

**Comment by jakirkham** [comment]
Once we bump to the new version, I think this should pass.

File: recipes/zwatershed/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/zwatershed/meta.yaml]: Is this version constraint accurate though? If not, we should definitely drop it.

**Comment by jakirkham** [comment]
I have placed a PR ( https://github.com/csinva/staged-recipes/pull/1 ) against your branch, @csinva, that should get this PR up and running. Please merge when you have a chance.

**Comment by jakirkham** [comment]
Thanks @csinva. This doesn't build on Windows. So, am not waiting for AppVeyor.

**Comment by jakirkham** [comment]
So you should get an invite to join conda-forge, @csinva. Once accepted you will be added to the `zwatershed` team and be given permissions over that feedstock (repo) and its CIs. When making an update, please fork the feedstock, make changes in your fork, and PR them back. This is very important as every branch on the original feedstock auto-deploys. Please let us know if you have any questions.

Welcome to conda-forge. 😄

**Pull request merged**